README.rst

```diff
@@ -26,6 +26,12 @@ This plugin requires `containers.podman` collection to be present:
 
 Please note that this driver is currently in its early stage of development.
 
+**Change podman executable**
+
+To change the podman executable from the standard podman, export environment variable MOLECULE_PODMAN_EXECUTABLE. For instance if you wish to run molecule with podman-remote instead of ordinary podman, the variable can be exportet as:
```

Comment:
```suggestion
To change the podman executable from the standard podman, export environment
variable ``MOLECULE_PODMAN_EXECUTABLE``. For instance, if you wish to run
molecule with ``podman-remote`` instead of ordinary ``podman``, the variable
can be exported as:
```